components/ProductImage/index.jsx

```diff
@@ -104,12 +109,11 @@ class ProductImage extends Component {
       );
     }
 
-    // Return the actual image.
     return (
       <Image
         {...this.props}
         backgroundColor={colors.light}
-        className={styles.container}
+        className={`${styles.container} ${this.props.styleUpdate.container}`}
```

Comment:
Please add checks here if we actually have a className, and only then add it here. Otherwise it will render 'null' (as a string) which is not clean.